### PR TITLE
Completion of dioalmaprotocol

### PR DESCRIPTION
**Pull request by DioAlma**
After 2 intense years and some days of development we finally concluded our project. This is the last pull request I intend to do in a while and maybe forever. Our project is composed of a total of 10 tokens (3+7) Dinheiros Reis and Escudos belong to all 7 other families and each one has their own token, Alem, Danu, AMG, ARK, TRS, PLASMA and DMT. This PR aims at completing the info for ALEM DANU TRS PLASMA AND DMT and it also includes some house keeping like removing duplicate or un used files. Thank you so much for everything so far keep up the amazing work and, ofcourse: KEEP PUSHING!

Much love,
Cpt. DioAlma Velveteen

### CODE REVIEW

File: terra2/assetlist.json [hunk 1]
```diff
@@ -247,7 +247,7 @@
       "base": "cw20:terra1cmf8ytutcwrjrv08zskj9phuh46a3w3nkjax7en4hxezsrdr58lqvzy05q",
       "name": "alentejo.money",
       "display": "Alem",
-      "symbol": "ALEM",
+      "symbol": "Alem",
```
Comment (JeremyParish69) [inline terra2/assetlist.json:250]: Please keep symbols uppercase

File: terra2/assetlist.json [hunk 2]
```diff
@@ -260,6 +260,186 @@
...
+      "base": "cw20:terra1en42e2vsvtdsvrcqg5s5e5e4djejjaed6fxmvtpweg3tu33h6k5qkg9c40",
+      "name": "almagem",
+      "display": "AMG",
+      "symbol": "AMG",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra2/images/almagem.png"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra2/images/almagem.png",
```
Comment (JeremyParish69) [inline terra2/assetlist.json:286]: Please keep image file size below 250KB

File: terra2/assetlist.json [hunk 3]
```diff
@@ -260,6 +260,186 @@
...
+      "base": "cw20:terra1ysd87nayjuelxj4wvp4wnp9as0mwszzkje6a9z6f3xx2903ghnsq4hm50y",
+      "name": "arika",
+      "display": "ARK",
+      "symbol": "ARK",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra2/images/arika.png"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra2/images/arika.png",
```
Comment (JeremyParish69) [inline terra2/assetlist.json:316]: Please remove the extra padding from all images

**Review by DioAlma — COMMENTED** [review]
Thank you for letting me know, I sent the modified images.

Best regards,
Cpt. DioAlma Velveteen

**Pull request merged**